src/index.html

```diff
@@ -1,14 +1,40 @@
 <!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Moyo header</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div class="container">
+      <a class="logo" href="/">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav__box">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="is-active" href="#">apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="is-active" href="#">apple</a>
          </li>
```

You can format your code like this to improve readability

Reply:
I fight with prettier for long time. i don`t know how to prove it.

Reply:
you can just ask for help in `fe_chat`